libs/vectorise/include/vectorise/fixed_point/fixed_point.hpp

```diff
@@ -670,11 +699,15 @@ constexpr FixedPoint<I, F>::FixedPoint(T n, meta::IfIsInteger<T> * /*unused*/)
 {
   if (CheckOverflow(static_cast<NextType>(n)))
   {
+    std::cout << "this = " << FromBase(data_) << std::endl;
```

Comment:
I don't think we want to do this in release mode.
there are cases we overflow is recoverable, and printing to console probably shouldn't be done in release unless it's already a catastrophic failure

Reply:
accidentally left out, removed.